### PR TITLE
Github Actions workflow to require tests to pass before allowing merge

### PR DESCRIPTION
**Pull request by yenaing-oo**
Closes https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/266

### CODE REVIEW

**Comment by yenaing-oo** [comment]
@danloveg 
Do we want to check that the tests pass on different Python versions?

**Comment by danloveg** [comment]
> @danloveg Do we want to check that the tests pass on different Python versions?

3.9 and 3.10 make sense to me. The container version is 3.10 which is what we really need to test.

File: .github/workflows/django-tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Django Tests
```
[the remark this answers is not in the document]
Comment (yenaing-oo) [reply .github/workflows/django-tests.yml]: Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/276/commits/b6da0addd01fab0d19088fe0cffc4220c15ee871

**Comment by yenaing-oo** [comment]
> This is awesome, it'll be super useful to have this in the future. I guess the `test:` block would output a failure when any of the steps in `steps:` fails? I'm just trying to get a sense of how it works

Yes, if any step in `steps:` returns a non-zero exit code, the entire workflow is marked as fail

**Review by danloveg — APPROVED** [review]

**Pull request merged**